homeworks/nagynorbert/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,70 @@
+# homework - Nagy Norbert
+
+# define numbers of rounds
+while True:
```

Comment:
ez a while blokk is lehetne egy function, pl `get_rounds()`, ami visszaadja hogy hány kört fognak játszani.